### ISSUE TITLE
angle after gaussian filter

### ISSUE THREAD

**Issue by jxtxwsbn** [skumra/robotic-grasping#42]
Hi, thanks for your code. The predicted angle image is passed into a gaussian filter. I think the value of the angle will be changed after that. Why need it? Thanks

**Comment by skumra** [comment]
Good point, the gaussian filter is used to mostly filter out the quality function. It probably shouldn't be applied to the angle.

Please feel free to open a pull request to fix this issue.

**Comment by jxtxwsbn** [comment]
Thanks. Maybe it doesn't affect too much since the results look good and the Gaussian filter just smooths the image a little.
